console/src/main/java/com/alibaba/nacos/console/controller/UserController.java

```diff
@@ -161,11 +161,11 @@ private boolean hasPermission(String username, HttpServletRequest request) {
         if (!authConfigs.isAuthEnabled()) {
             return true;
         }
-        if (Objects.isNull(request.getAttribute(RequestUtil.NACOS_USER_KEY))) {
+        if (Objects.isNull(request.getSession().getAttribute(RequestUtil.NACOS_USER_KEY))) {
```

Comment:
this commit is that when the authentication switch is turned on, there is no way to change the password(forbidden), because the user information has been saved in the session, but the previous code was obtained directly from reqeust.